discord/guild.py

```diff
@@ -3931,6 +3933,36 @@ async def widget(self) -> Widget:
 
         return Widget(state=self._state, data=data)
 
+    def widget_image_url(self, style: WidgetStyle = 'shield') -> Optional[str]:
+        """
+
+        Returns the widget image url of the guild.
+
+        .. note::
+
+            The guild must have the widget enabled to get this information.
+
+        Parameters
+        -----------
+        style: :class:`str`
+            The style which should be applied for the widget.
+            Default to ``shield``. Returns ``None`` if widget is not enabled.
+
+        Raises
+        -------
+        HTTPException
+            Retrieving the widget image url failed.
```

Comment:
This doesn't raise anything.

Reply:
Ah okay. Well it does when the user specifies a wrong `style` but that's not in the bot's scope anymore, the link just returns an error in the browser.